Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -1347,6 +1349,25 @@ theorem isBridge_iff_mem_and_forall_cycle_not_mem {e : Sym2 V} :
     G.IsBridge e ↔ e ∈ G.edgeSet ∧ ∀ ⦃u : V⦄ (p : G.Walk u u), p.IsCycle → e ∉ p.edges :=
   Sym2.ind (fun _ _ => isBridge_iff_adj_and_forall_cycle_not_mem) e
 
+/-- Deleting a non-bridge edge from a connected graph preserves connectedness.
+TODO : Iff version. -/
+lemma Connected.connected_delete_edge_of_not_isBridge (hG : G.Connected) {x y : V}
+    (h : ¬ G.IsBridge s(x, y)) : (G \ fromEdgeSet {s(x, y)}).Connected := by
```

Comment:
There is a definition for edge deletion which I think we should use (I note the lemmas above do not but I claim they should):
```suggestion
lemma Connected.connected_deleteEdge_of_not_isBridge (hG : G.Connected) {x y : V}
    (h : ¬ G.IsBridge s(x, y)) : (G.deleteEdges {s(x, y)}).Connected := by
```
(you'll need to add `rw [deleteEdges]` or similar to fix the proof)